htdocs/xoops_lib/Xoops/Core/Assets.php

```diff
@@ -96,7 +96,7 @@ class Assets
      * @var string string to identify Assetic filters using instanceof
      */
 
-    private $filterInterface = '\Assetic\Filter\FilterInterface';
+    private $filterInterface = '\\Assetic\\Filter\\FilterInterface';
```

Comment:
Le 24/09/2015 19:02, Richard Griffith a écrit :
> In htdocs/xoops_lib/Xoops/Core/Assets.php
> <https://github.com/XOOPS/XoopsCore/pull/404#discussion_r40343942>:
>
>> @@ -96,7 +96,7 @@ class Assets
>>       * @var string string to identify Assetic filters using instanceof
>>       */
>>
>> -    private $filterInterface = '\Assetic\Filter\FilterInterface';
>> +    private $filterInterface = '\\Assetic\\Filter\\FilterInterface';
>
> Why are we doubling up the backslashes? It is needed for double quoted
> strings, but not in single quoted ones.
>
> A single quote needs to be escaped, so if there is a backslash at the
> end of the string, that one needs to be escaped, i.e.
> |'\Assetic\Filter\\' . $filtername|, but otherwise they are not
> required. The '\' works, but make searching for string references to
> classes much more complicated than it should be.
>
> Is there an issue I've missed that would make these required?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/XOOPS/XoopsCore/pull/404/files#r40343942>.
>


You are enterely right.

I would double backslash to avoid confusing between sigle or double 
qouted strings. But I is not required.